docs/profiler/application-capability-profiler.md

```diff
@@ -60,9 +60,11 @@ Module: [Microsoft.Windows.Win32Isolation.ApplicationCapabilityProfiler](referen
 In administrator PowerShell 7:
 
 ```PowerShell
-Import-Module Microsoft.Windows.Win32Isolation.ApplicationCapabilityProfiler.dll
+Import-Module .\Microsoft.Windows.Win32Isolation.ApplicationCapabilityProfiler.dll
 ```
 
+*Note:* the Microsoft.Windows.Win32Isolation.ApplicationCapabilityProfiler.dll module is located in the ACP folder extracted from the archive in step 0.5 above.

```

Comment:
`Step 0.5` is a bit confusing as we do not really have a section 0. We can put a link there

```
from the archive in the [step](link) above.
```

Reply:
There is a profiling step 0 above. I've included a link anyway.